src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheckTest.java

```diff
@@ -31,6 +33,21 @@
     extends BaseCheckTestSupport {
 
     @Test
+    public void testStaticVisibilityFromLambdas() throws Exception {
+        final DefaultConfiguration checkConfig =
+            createCheckConfig(HiddenFieldCheck.class);
+        final String[] expected = {
+            "16:34: " + getCheckMessage(MSG_KEY, "value"),
+            "48:23: " + getCheckMessage(MSG_KEY, "languageCode"),
+            "57:35: " + getCheckMessage(MSG_KEY, "num1"),
+            "70:35: " + getCheckMessage(MSG_KEY, "num2"),
+            "98:23: " + getCheckMessage(MSG_KEY, "lc1"),
```

Comment:
can we use here some simple words like `variable`, `parameter`, instead of cryptic `lc1`?